mmdet/core/post_processing/matrix_nms.py

```diff
@@ -0,0 +1,58 @@
+import torch
+
+
+def matrix_nms(seg_masks, cate_labels, cate_scores,
+               kernel='gaussian', sigma=2.0, sum_masks=None):
+    """Matrix NMS for multi-class masks.
+
+    Args:
+        seg_masks (Tensor): shape (n, h, w)
+        cate_labels (Tensor): shape (n), mask labels in descending order
+        cate_scores (Tensor): shape (n), mask scores in descending order
+        kernel (str):  'linear' or 'gauss'
```

Comment:
'linear' or 'gaussian'. The docstring should be consistent with the argument.

Reply:
fixed.:)